packages/next/src/build/webpack-config.ts

```diff
@@ -1974,6 +1974,14 @@ export default async function getBaseWebpackConfig(
           )
         ),
     ].filter(Boolean as any as ExcludesFalse),
+    ignoreWarnings: [
+      (warning) => {
```

Comment:
should we also test the module is `react-dom`?

Reply:
Don't know how to do that.

Reply:
warning is a WebpackError instance, it also has a `module` instance, and we could check the resource of module, and matching `warning.module.resource`? but not blocking, I guess the message is quite unique for now.

Reply:
I think I checked that but couldn't find the relevant string. I'll follow up with removing the try-catch anyway and then this block will disappear.

Reply:
Fixed in https://github.com/vercel/next.js/pull/70463